### ISSUE TITLE
Unable to remove chips when Talkback is enabled on Android

### ISSUE THREAD

**Issue by gabrielarcos** [angular/components#12351]
#### Bug:
Unable to remove chips from chip list when Talkback tool is enabled on Android 7.0 

#### What is the expected behavior?
Should be able to remove chip from chip list by tapping on the remove icon.

#### What is the current behavior?
The chip gets active but no way to remove

#### What are the steps to reproduce?
On the Android device go to Settings -> Accessibility, tap on Talkback and turn it on. Go to Chrome for Android an open the demo site https://material.angular.io/components/chips/overview scroll down to "Chip input example" and try to remove one of the chips

**Comment by Splaktar** [comment]
Need to test this against the latest Angular Material and against MDC-based chips.

**Comment by zarend** [comment]
Hello @gabrielarcos ,

Thank you for reporting this issue. I believe it has been fixed in v15. Does the issue still happen for you?

Thanks,
Zach

**Comment by amysorto** [comment]
Closing this issue due to no recent activity, if this is still an issue please create a new issue with a reproduction.
